test/unittest/dds/participant/ParticipantTests.cpp

```diff
@@ -191,6 +192,29 @@ TEST(ParticipantTests, CreateDomainParticipantWithProfile)
     ASSERT_TRUE(DomainParticipantFactory::get_instance()->delete_participant(participant) == ReturnCode_t::RETCODE_OK);
 }
 
+TEST(ParticipantTests, GetParticipantProfileQos)
+{
+    DomainParticipantFactory::get_instance()->load_XML_profiles_file("test_xml_profiles.xml");
+    DomainParticipantQos qos;
+    ASSERT_EQ(
+        DomainParticipantFactory::get_instance()->get_participant_qos_from_profile("test_participant_profile", qos),
+        ReturnCode_t::RETCODE_OK);
+
+    // Extract ParticipantQos form profile
+    DomainParticipant* participant =
+            DomainParticipantFactory::get_instance()->create_participant(0, qos);
+    ASSERT_NE(participant, nullptr);
+
+    check_participant_with_profile(participant, "test_participant_profile");
+
+    // Clean up
```

Comment:
This comment should be after this instruction. Here something like
`// Test return when a non-existent profile is used`
should be.